bundle/manifests/node-healthcheck-operator.clusterserviceversion.yaml

```diff
@@ -389,7 +392,7 @@ spec:
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.namespace
-                image: quay.io/medik8s/node-healthcheck-operator:latest
+                image: quay.io/slintes/node-healthcheck-operator:latest
```

Comment:
this is probably the leftover of a test :)

Reply:
absolutely 🙈